plugins/esi/esi.cc

```diff
@@ -837,8 +837,8 @@ transformData(TSCont contp)
     CONT_DATA_DBG(cont_data, "[%s] trying to process doc", __FUNCTION__);
     string                   out_data;
     string                   cdata;
-    int                      overall_len;
-    EsiProcessor::ReturnCode retval = cont_data->esi_proc->flush(out_data, overall_len);
+    int                      overall_len = 0;
```

Comment:
This variable was used uninitialized before later in this function. The assumption was that `flush` would set the value to some value, always. Even in error cases. Before this patch it wasn't. We now initialize the variable to 0.